### ISSUE TITLE
remove unused AO CreateAsyncIteratorFromClosure

### ISSUE THREAD

**Issue by michaelficarra** [tc39/ecma262#3509]
It's unused. We should simply delete it.

**Comment by ljharb** [comment]
It was added in #2045, in the pursuit of built-in generators, for async iterator helpers. Will async iterator helpers use it? If so, it's probably best not to create more churn by removing it; if not, it should ofc be axed.

**Comment by bakkot** [comment]
> Will async iterator helpers use it?

No, since the concurrency stuff they're trying to do doesn't work with generators.
